code/datums/craft/recipes/weapon.dm

```diff
@@ -217,7 +216,6 @@
 		list(QUALITY_ADHESIVE, 10, "time" = 5),
 		list(/obj/item/stack/rods, 1),
 		list(QUALITY_ADHESIVE, 10, "time" = 5),
-		list(/obj/item/stack/rods, 1),
```

Comment:
This weapon is three rods taped together with a blade (both sprite and desc-wise)

Reply:
Oh thats my bad. I found the crafting recipe annoying so I wanted to change it, but forgot to actually make it still cost 3 rods.

Reply:
Upon reflection this recipe was fine and i just changed it back to normal

Reply:
You could make it 2 steps
Step 1 put together all 3 rods at once
Step 2 adhesive

Reply:
I did that for the halberd since it has more steps already than the staff, but I left the staff as is. Having a 2 step crafting recipe felt kinda... not right? I'm not sure how to explain it. Either way I'm not particularly bothered by the original recipe.

Reply:
To make halberd recipe better you can make it just ask for the makeshift staff, but this will eat any mods installed too if people for some reason use a modded one.

Reply:
I like that a lot better. It makes the staff a lot more useful since you can upgrade it later when you get better tools.